web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss

```diff
@@ -22,12 +22,15 @@
     }
 
     &.usa-current {
-      @include vote-link-bg($color: $base-dark);
+      // @include vote-link-bg($color: $base-dark);
+      @include vote-link-bg; // temporarily using default color
 
+      // section indicator (left of current section heading)
       &:after {
         @include u-left(0);
         inset-inline-start: 0;
         background-color: $base-dark;
+        visibility: hidden; // temporarily hiding section indicator
       }
```

Comment:
i would use `display: none;` and add it to top of the list, can even comment out the rest to show that it is supposed to be turned off.